src/Symfony/Bundle/WebProfilerBundle/Tests/DependencyInjection/WebProfilerExtensionTest.php

```diff
@@ -31,6 +31,9 @@ public static function assertSaneContainer(Container $container, $message = '')
     {
         $errors = array();
         foreach ($container->getServiceIds() as $id) {
+            if ('service_container' === $id) { // to be removed in 4.0
```

Comment:
Not sure.. wouldnt this be over complex? <4.0 `getServiceIds` includes privates, using those with `get()` etc. is non-crashing, in >=4.0 those id's are exlcuded as they are crashing. I could live with that.

For the SF codebase i think explicitly skippping is just fine =/